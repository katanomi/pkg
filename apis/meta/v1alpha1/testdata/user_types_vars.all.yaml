kind: "ServiceAccount"
apiGroup: "v1"
name: "default"
namespace: "default"
