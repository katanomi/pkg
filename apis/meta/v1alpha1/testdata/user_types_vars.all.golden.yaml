user: ""
user.kind: "ServiceAccount"
user.apiGroup: "v1"
user.name: "default"
user.namespace: "default"
